src/db/oracledb/phones-dao.js

```diff
@@ -1,20 +1,24 @@
+import _ from 'lodash';
+
 import { parseQuery } from 'utils/parse-query';
 import { getConnection } from './connection';
 import { contrib } from './contrib/contrib';
 
 /**
  * Queries data source for raw person data and passes it to the serializer
  *
- * @param {string} osuId OSU ID of person to select
+ * @param {string} pidm PIDM of person to select
  * @param {object} query query parameters passed in with the request
  * @returns {Promise<object>} Raw phone data from data source
  */
-const getPhones = async (osuId, query) => {
+const getPhones = async (pidm, query) => {
   const connection = await getConnection();
   try {
     const parsedQuery = parseQuery(query);
```

Comment:
`filter[primaryInd]=foo` is considered as a valid parameter which shouldn't be. We need to add validation logic to only allow boolean.